Aptoide/aptoide-client-v8
# [DT-47] MyApps step 2

**What does this PR do?**

   This PR aims at implementing the step 2 of my apps bundle, which shows the list of installed apps and consequent more bundle view.

Reference to the other pull request: [DT-47](https://github.com/Aptoide/aptoide-client-dt/pull/28)

Note: limited the main bundle to 9 apps as it is the same limit as we have on other bundles. This was not pre-defined.

**Database changed?**

   No

**Where should the reviewer start?**


**How should this be manually tested?**

  Open the app, press allow on the bundle, check the apps, check the apps in the more view, try to open them.

**What are the relevant tickets?**

  Tickets related to this pull-request: [DT-47](https://aptoide.atlassian.net/browse/DT-47)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new keys, etc.) 



**Code Review Checklist**
